systemtest/src/main/java/org/bf2/systemtest/operator/FleetShardOperatorManager.java

```diff
@@ -56,9 +56,12 @@ public static CompletableFuture<Void> deployFleetShardOperator(KubeClient kubeCl
         }
         printVar();
         LOGGER.info("Installing {}", OPERATOR_NAME);
-
-        installedCrds =
-                Files.list(CRD_PATH).filter(p -> p.getFileName().toString().endsWith(CRD_FILE_SUFFIX)).collect(Collectors.toList());
+        Stream<Path> credFilePath = Files.list(CRD_PATH);
+        try {
```

Comment:
This can be simplified further using try-with-resource by placing `Stream<Path> credFilePath = Files.list(CRD_PATH)` in the `try (...)` and removing the `finally` block.